### ISSUE TITLE
Product cannot show in category pages

### ISSUE THREAD

**Issue by radhagobinda** [magento/magento2#6174]
### Preconditions

1.  I am using magento 2 with php 7
### Problem

Products cannot show in my category pages. Actually I shifted my code one server to another server. Then problem was started. I did everything. 
a> Content deploy
b> Cache clear
c> Re-indexing

In admin panel, it shows but problem in front end category pages. Home page is good. 
You can check my website : http://fansmart.codeuridea.net/ 

If any one knows about this problem, please share it. 
### Actual Result

![screenshot from 2016-08-17 11 18 11](https://cloud.githubusercontent.com/assets/19976130/17725859/70273200-646c-11e6-942b-60b202a61b8a.png)
### Expected Result

![screenshot from 2016-08-17 11 17 01](https://cloud.githubusercontent.com/assets/19976130/17725874/8c8c8e18-646c-11e6-8376-cbf5af4e7ba7.png)

**Comment by palamar** [comment]
Please, format this issue according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines): 
- with steps to reproduce
- actual result 
- expected result

**Comment by radhagobinda** [comment]
@palamar Please check that.

**Comment by radhagobinda** [comment]
Above I mention that I shifted my code one server to another server and domain also be changed. 

Today I realize that if I save a product through admin panel then this particular product shows in category page. But problem is that I have more than 1 lac product and it is impossible to save individually. 

So Please help me how to show my all products in category pages.

**Comment by palamar** [comment]
Thanks for reporting this issue, but without detailed steps to reproduce:
- detailed version of Magento 2 (2.0.0, 2.0.8, 2.1.0 etc.)
- how migration was performed (import/export, MySql dump, 3rd party modules etc.)
  we can't reproduce your issue and decide is it possible to fix it

P.S. 
if save of the product resolves you issue, you can try to use mass action as a workaround (eg. unassign/assign all products from/to website)

**Comment by radhagobinda** [comment]
1> Magento Version: 2.02
2> I moved all files from old server to new server. Then import the database and change the base url in config table. After all this action I did content deploy, re-index, cache clear through console also I did file permission for var and pub folder.

After all this actions, site is running again. All pages are running, but only products are not showing in category pages.
